ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/Constants.java

```diff
@@ -38,6 +38,8 @@ public class Constants {
 
   public static final Duration ETH1_INDIVIDUAL_BLOCK_RETRY_TIMEOUT = Duration.ofMillis(500);
   public static final Duration ETH1_DEPOSIT_REQUEST_RETRY_TIMEOUT = Duration.ofSeconds(2);
+  public static final Duration EL_REQUEST_TIMEOUT = Duration.ofSeconds(2);
```

Comment:
This is incorrect.  The execution timeout is 2 *minutes* not 2 seconds.

Reply:
My fault. And it's an answer on my second question. Will fix